packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js

```diff
@@ -310,145 +280,180 @@ class SignTxPage extends Component<Props, State> {
-                          amount: txData.total.get(
-                            txData.amount.defaults.defaultIdentifier
-                          ) ?? (new BigNumber('0')),
-                        },
-                      })}
-                    </div>
-                  </div>
-                </div>
-                <div className={styles.passwordInput}>
-                  <TextField
-                    type="password"
-                    className={styles.walletPassword}
-                    {...walletPasswordField.bind()}
-                    error={walletPasswordField.error}
-                  />
-                </div>
-                <div className={styles.wrapperBtn}>
-                  <Button
-                    variant="secondary"
-                    className="secondary"
-                    onClick={onCancel}
-                  >
-                    {intl.formatMessage(globalMessages.cancel)}
-                  </Button>
-                  <LoadingButton
-                    variant="primary"
-                    disabled={!walletPasswordField.isValid}
-                    onClick={this.submit.bind(this)}
-                    loading={isSubmitting}
-                  >
-                    {intl.formatMessage(globalMessages.confirm)}
-                  </LoadingButton>
-                </div>
-              </div>
-            ) : (
-              <CardanoUtxoDetails
-                txData={txData}
-                onCopyAddressTooltip={this.props.onCopyAddressTooltip}
-                addressToDisplayString={this.props.addressToDisplayString}
-                getCurrentPrice={this.props.getCurrentPrice}
+      <SignTxTabs
+        overviewContent={
+          <Box paddingTop="8px" overflowWrap="break-word">
+            <Typography color="var(--yoroi-palette-gray-900)" variant="h5" marginBottom="8px">
+              {intl.formatMessage(signTxMessages.connectedTo)}
+            </Typography>
+            <Box
+              display="flex"
+              alignItems="center"
+              px="28px"
+              py="20px"
+              border="1px solid var(--yoroi-palette-gray-100)"
+              borderRadius="6px"
+              minHeight="88px"
+              mb="8px"
+            >
+              {faviconUrl != null && faviconUrl !== '' ? (
+                <Box
+                  sx={{
+                    marginRight: '12px',
+                    width: '32px',
+                    height: '32px',
+                    border: '1px solid #a7afc0',
+                    borderRadius: '50%',
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    backgroundColor: '#f8f8f8',
+                    img: {
+                      width: '20px',
+                    },
+                  }}
+                >
+                  <img src={faviconUrl} alt={`${url} favicon`} />
+                </Box>
+              ) : null}
```

Comment:
We should display a placeholder image.We already have this in the connected websites screen in Yoroi 
[check this ](https://github.com/Emurgo/yoroi-frontend/blob/develop/packages/yoroi-extension/app/components/dapp-connector/ConnectedWebsites/WalletRow.js#L140)